aviary/utils/legacy_code_data/flops_default_values.txt

```diff
@@ -1,5 +1,14 @@
 Default Values from FLOPS
 
   &WTIN
-  EEXP=1.15,
+  EEXP=1.15,unitless
+  IALTWT=False,unitless
+  CARGF=False,unitless
+  CARBAS=False,unitless
+  &ENGDIN
```

Comment:
the `\` is used to terminate each NAMELIST
```suggestion
 \
  &ENGDIN
```

Reply:
Done.